src/data_processing/data_processing_and_training_demo.ipynb

```diff
@@ -0,0 +1,1559 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "source": [
+    "# Data Preprocessing\n",
+    "#### This notebook shows the process to extract the ppg, ecg, and blood pressure (abp) data from the .mat file, convert the data to time series and put them into a dataloader for training the LSTM. A training demo is also included to show that the dataloader works."
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "outputs": [],
+   "source": [
+    "import matplotlib.pyplot as plt\n",
+    "import numpy as np\n",
+    "import h5py\n",
+    "import os\n",
+    "\n",
+    "import torch\n",
+    "import torch.nn as nn"
+   ],
+   "metadata": {
+    "collapsed": false
+   }
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 103,
+   "outputs": [],
+   "source": [
+    "# !!!! If you want to run this notebook, change the file path\n",
+    "file_path = \"G:/PycharmProjects/APS360Project/data/Part_1.mat\"\n",

```

Comment:
maybe create something like a private_data.ini file where each of us can store our paths to make it easier and read from that file to set the file path

Reply:
or use the os function chdir to do something like os.chdir(os.path.dirname(os.path.realpath(__file__))) to get the current directory and set file_path to '../../data/Part_1.mat'